# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/blender/python-tcl-tk-dep.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./SConstruct.orig	2004-12-18 04:32:59.000000000 +0100
+++ ./SConstruct	2004-12-18 04:45:58.000000000 +0100
@@ -113,7 +113,7 @@
 	python_lib = ['python%d.%d' % sys.version_info[0:2]]
 	python_libpath = [sysconfig.get_python_lib (0, 1) + '/config']
 	python_include = [sysconfig.get_python_inc ()]
-	python_linkflags = Split (sysconfig.get_config_var('LINKFORSHARED'))
+	python_linkflags = Split ("-ltcl8.4 -ltk8.4 "+sysconfig.get_config_var('LINKFORSHARED'))
 	# International support information
 	ftgl_lib = ['ftgl']
 	ftgl_libpath = ['#../lib/linux-glibc2.2.5-i386/ftgl/lib']
